api/src/main/java/org/pmiops/workbench/workspaceadmin/WorkspaceAdminController.java

```diff
@@ -163,19 +49,8 @@ public WorkspaceAdminController(
       Integer limit,
       Long afterMillis,
       @Nullable Long beforeMillisNullable) {
-    final long workspaceDatabaseId =
-        workspaceAdminService
-            .getFirstWorkspaceByNamespace(workspaceNamespace)
-            .map(DbWorkspace::getWorkspaceId)
-            .orElseThrow(
-                () ->
-                    new NotFoundException(
-                        String.format(
-                            "No workspace found with Firecloud namespace %s", workspaceNamespace)));
-    final DateTime after = new DateTime(afterMillis);
-    final DateTime before =
-        Optional.ofNullable(beforeMillisNullable).map(DateTime::new).orElse(DateTime.now());
     return ResponseEntity.ok(
-        actionAuditQueryService.queryEventsForWorkspace(workspaceDatabaseId, limit, after, before));
+        workspaceAdminService.getAuditLogEntries(
```

Comment:
one could argue that the `WorkspaceAdminController` should depend on the `ActionAuditQueryService` directly instead of putting `getAuditLogEntries()` here.